src/firmware/main/primitives/accurate_shoot.c

```diff
@@ -188,13 +187,28 @@ static void accurate_shoot_tick(log_record_t *log)
     }
     else
     {
+        FirmwareRobot_t* robot = app_firmware_world_getRobot(world);
+
         // accelerate at ball to kick it
-        chicker_auto_arm((global_params->extra & 1) ? CHICKER_CHIP : CHICKER_KICK,
-                         global_params->params[3]);
-        if (!(global_params->extra & 1))
+        Chicker_t* chicker = app_firmware_robot_getChicker(robot);
+        bool chipping      = global_params->extra & 1;
```

Comment:
Nit:
Would be nice to have a CHIPPING_MASK instead of a '1' here

Reply:
I'm gonna say that's out of scope for now, this will naturally get fixed when we transition to protobuf.